test/integration/targets/azure_rm_appserviceplan/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+posix/ci/cloud/group2/azure
```

Comment:
The aliases have just recently changed, have a look at https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_loadbalancer/aliases to see how to name this now.

Reply:
fixed